Data-Structures/src/main/java/Hashtable/Hashtable.java

```diff
@@ -0,0 +1,87 @@
+package Hashtable;
+
+import java.util.HashMap;
+
+public class Hashtable {
+
+    public Llist[] arr;
+    public int size;
+
+
+    public void setArr(Llist[] arr) {
+        this.arr = arr;
+    }
+
+    public void setSize(int size) {
+        this.size = size;
+    }
+
+    //constructor
+    public Hashtable(){
+        size=50;
+        arr=new Llist[size];
+    }
+
+    public Hashtable(int size){
+        this.size=size;
+        arr=new Llist[size];
+    }
+
+    //add value to hashtable
+    public void add(String key,String value){
+   int index=hash(key);
+        if (arr[index] == null)
+        {
+            arr[index] = new Llist();
+        }
+        arr[index].add(key, value);
+
+
+    }
+
+    //hash the key
+    public int hash(String key) {
+        int hashValue = 0;
+        char[] letters = key.toCharArray();
+        for (int i = 0; i < letters.length; i++) {
+            hashValue += letters[i];
+        }
+        hashValue = (hashValue * 599) % arr.length;
+        return hashValue;
+    }
+
+    //delete item from hashtable
+    public boolean delete(String key)
+    {
+        int index = hash(key);
+        if (arr[index] != null)
+        {
+            return arr[index].delete(key);
+        }
+        return false;
+    }
+
+    public Llist get(String key)
+    {
+        int index = hash(key);
+        if (arr[index] != null)
+        {
+            return arr[index];
+        }
+
+        return null;
+    }
+
+    public boolean contains(String key)
```

Comment:
This method won't work properly if there was a collision. If arr[index] has a LinkedList then this method will return true even if the actual key that's passed in isn't there.